### DISCUSSION TITLE
Add reward for completing story mode 100%

### DISCUSSION THREAD

**Discussion by Ordoviz** [SuperTux/supertux#2116] [Ideas]
I suggest adding a little reward text that appears when you complete every level in the story mode 100%, meaning you collected every coin, killed all (killable) badguys, found every secret and beat the levels in time. The reward doesn’t need to be _much_, it should just be _something_ because I got a bit disappointed when I achieved 100% and got _nothing_ other than level dots blinking around.

There could be possibly a 100% trophy image with “Congratulations!” displayed over the worldmap.

Having played Undertale I would also like to add these lines in red color:

> …but there’s no one left who could celebrate your achievement with you
> …because you killed every being in this world.
> In addition, you have stolen all gold from them to enrich yourself.
> What kind of ruthless freak are you?

Then the screen fades to black and the player is prompted to the title menu. The player can then unlock an evil sprite for Tux.

_(The last things are optional)_

**Comment by Rusty-Box** [comment]
This is actually a pretty nice idea. (Except for the last part :D)
However this might have to wait for a future release.

**Comment by Alasdairbugs** [comment]
I like the idea of being rewarded, but maybe instead of an evil tux, maybe a tux with a crown on his head? The idea of copying undertale's morality theme is silly though. Undertale is an RPG, supertux isn't. It's just a platformer. Copying it would be so.. unoriginal.
